app/models/raw_input_changes.rb

```diff
@@ -10,7 +10,7 @@ def initialize(raw_input)
     @attrs = raw_input.transform
     @organization = nil
     @relations = {}
-    @relations_to_build = [:email, :location, :organization_name, :phone_number]
+    @relations_to_build = %i(email location organization_name phone_number)
```

Comment:
I am not super sure about this rule (as I am very used to the `:symbol` syntax), but this does shorten the length.